unic/char/property/src/macros.rs

```diff
@@ -48,13 +48,38 @@
 /// # fn main() {}
 /// ```
 ///
+/// # Syntax (Binary Property)
+///
+// No compile because the included file does not exist
+/// ```ignore
+/// #[macro_use] extern crate unic_char_property;
+///
+/// // First we define the type itself.
+/// char_property! {
+///     /// This is the newtype used for the character property.
+///     pub struct MyProp(bool) {
+///         abbr => "AbbrPropName";
+///         long => "Long_Property_Name";
+///         human => "Human-Readable Property Name";
+///     }
+///
+///     /// Unlike an enumerated property, a binary property will handle the table for you.
+///     /// This requires `unic_utils` to be in scope.
+///     mod data = "path/to/table.rsv";
```

Comment:
Since the `data` module is not exposed (and in this case, since it's auto-created, we can actually skip it), I think this one is better as something like:
```
data_table => "path/to/table.rsv";
```
similar to setting name strings.

What do you think?